pyERGM/utils.py

```diff
@@ -1005,3 +1005,19 @@ def sample_from_independent_probabilities_matrix(probability_matrix, sample_size
             sample[i, j, :] = np.random.binomial(1, probability_matrix[i, j], size=sample_size)
 
     return sample
+
+
+# TODO: njit?
```

Comment:
why not?

Reply:
Just haven't tried to speedup everything yet. I thought it's better to see whether it works and worth the effort at first. 
Anyways, it runs by construction only once per call to fit, so I'm not sure that the time saved will be larger than the compilation time added.